### ISSUE TITLE
Override global rules from in-source attributes?

### ISSUE THREAD

**Issue by Fishrock123** [ericseppanen/cargo-cranky#6]
Similar to what was mentioned in https://github.com/ericseppanen/cargo-cranky/issues/3#issue-1298199372, but a little different. Perhaps that issue exists because this is just impossible.

I'd like to be able to use this tool, but with a blanket set of fairly "aggressive" rules across an entire organization.

Sometimes, like in the case of `clippy::print_stdout` this is appropriate for most projects (encourage folks to use the `log` crate), but I may want to disable it fro a CLI tool. Ideally I'd like to not have to adjust the project or workspace level `Cranky.toml` (since it may be automated). So, i'm hoping there would be a way to set an override from an in-code attribute, since that seems the most reasonable place to have such a thing...

See also https://rust-lang.zulipchat.com/#narrow/stream/257328-clippy/topic/Have.20in-code.20lint.20attributes.20have.20higher.20precidence.20than.20.2E.2E.2E/near/291373867

**Comment by jsenkpiel-godaddy** [comment]
Looks what I am experience is likely a clippy bug for specific rules: https://github.com/rust-lang/rust-clippy/issues/6610

**Comment by Fishrock123** [comment]
Looks what I am experience is likely a clippy bug for specific rules: https://github.com/rust-lang/rust-clippy/issues/6610

**Comment by ericseppanen** [comment]
If `Cranky.toml` had a way to say "load rules from this other toml file first", would that be useful?

**Comment by Fishrock123** [comment]
Yes, but I'd rather do this finer grained stuff in the source code - sometimes we may not have an extra directory for a second file to live in.

**Comment by ericseppanen** [comment]
cargo-cranky is a very thin wrapper around cargo-clippy, so it's pretty limited in what it can do. If it's not possible on the clippy commandline, then I don't think it's possible to fix from here.
